alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -159,9 +173,6 @@ export class BoardComponent implements OnInit, OnDestroy {
     if (boardCode === BOARD_TYPE_CODES.VETT) {
       this.createCardTitle = '+ New Application';
       this.cardDialogType = CreateApplicationDialogComponent;
-    } else {
```

Comment:
I see this was deleted but I don't see anything replacing it, can we still create recons on boards except vett?

Reply:
yes, the part for dialog is in html